jfrog-cli/artifactory/cli.go

```diff
@@ -1647,13 +1647,12 @@ func buildCollectEnvCmd(c *cli.Context) {
 	cliutils.ExitOnErr(err)
 }
 
-func buildAddGitCmd(c *cli.Context) {
+func buildAddGitCmd(c *cli.Context) error {
 	if c.NArg() > 3 || c.NArg() < 2 {
 		cliutils.PrintHelpAndExitWithError("Wrong number of arguments.", c)
 	}
 	buildAddGitConfiguration := createBuildAddGitConfiguration(c)
-	err := buildinfo.AddGit(buildAddGitConfiguration)
-	cliutils.ExitOnErr(err)
+	return buildinfo.AddGit(buildAddGitConfiguration)
```

Comment:
This is similar to what we do in goCmd.
This change is meant to prevent the tests from exiting in case of failure during the build-add-git command. That way, when an error occurs, the test performs a cleanup of the .git folder and after that it fails.